django/cantusdb_project/main_app/urls.py

```diff
@@ -283,8 +283,8 @@
     ),
     path(
         "source/<int:source_id>/chants/",
-        ChantListView.as_view(),
-        name="chant-list",
+        SourceBrowseChantsView.as_view(),
+        name="browse-chants",
```

Comment:
in `urls.py` we also have the `redirect-chant-list` path that could be renamed to `redirect-browse-chants`

Reply:
you're right! I should be able to fix this first thing on Thursday. Thanks!

Reply:
...actually, isn't the old path, the one we're redirecting from, "chant-list"? if that's the case, I think `redirect-chant-list` is a more accurate name.

But I should check in on this to make sure we don't have any out-of-date comment, at least. Again, thanks - this is a worthwhile thought!

Reply:
The old path is redirecting from `/chants/?source=<int:source_id>`. Anything with `/chant-list` brings a 404 so I don't believe this was a path that we set up a redirect for or that ever existed.

Reply:
I renamed `redirect_chant_list` to `redirect_chants` - it's not super descriptive, but the path was literally `/chants/`, so I feel it's accurate at least.